src/overlays/actors/ovl_En_Goroiwa/z_en_goroiwa.c

```diff
@@ -84,7 +84,15 @@ static ColliderJntSphInit sJntSphInit = {
 
 static CollisionCheckInfoInit sColChkInfoInit = { 0, 12, 60, MASS_HEAVY };
 
-static f32 sUnused[] = { 10.0f, 9.2f };
+static f32 sSpeeds[] = { 10.0f, 9.2f };
+
+#if OOT_DEBUG
+#define GET_EN_GOROIWA_SPEED(this) (R_EN_GOROIWA_SPEED * 0.01f)
+#define SET_EN_GOROIWA_SPEED(speed) R_EN_GOROIWA_SPEED = speed
+#else
+#define GET_EN_GOROIWA_SPEED(this) sSpeeds[(this)->isInKokiri]
+#define SET_EN_GOROIWA_SPEED(speed)
```

Comment:
Not a huge fan of the set macro between expanding to an assignment, or expanding to nothing
I'd rather see the assignment ifdefed out in code

Also the GET_ macro looks too much like the macros for getting data from actor params, but I don't have a suggestion

Reply:
Okay I remove the SET_ macro and left the GET_ macro alone for now

Reply:
fwiw I agree about the speed macro looking the a params macro, and think it would be fine as just "SPEED" without GET. If this isnt changed now, I assume it would be at some point in the future when the actor+params are documented. So either now or later is fine with me

Reply:
changed to `EN_GOROIWA_SPEED(this)`